### ISSUE TITLE
Main menu dropdown items not accessible on mobile

### ISSUE THREAD

**Issue by quaestor** [theCrag/website#3395]
**What happened?**

Clicking a main menu item (Index, Map, Climbers, Forum, Photos, Logbook, Help) on mobile does not open the dropdown.

It is therefore very hard to navigate to, e.g., your friends' ascents at a certain crag.

**What you expected:**

Clicking an item in the main menu opens a dropdown or an equivalent mechanism.

**Comment by quaestor** [comment]
No opinions on this? I feel this should be a priority, since it makes e.g. the help menu completely useless on mobile.

**Comment by brendanheywood** [comment]
I agree totally, it's just time and effort with lots of conflicting priorities

Would you prefer it to be a single tap to open the menu, and the tap a second time on the tab to open the main tab (ie the menu is not used at all), or would you prefer a down arrow next to each tab which when tapped opens the menu?  (which also spreads the tabs further apart)

**Comment by quaestor** [comment]
That's probably a matter of taste; my preference would be single tap to open, because little arrows are sometimes hard to hit and they additionally widen the menu so you need to scroll more.

Another option would be to open a fullscreen overlay menu like https://www.w3schools.com/howto/howto_js_fullscreen_overlay.asp when tapping a menu item.

Yet another option would be to incorporate the dropdowns into the hamburger menu.

**Comment by brendanheywood** [comment]
I've considered many options. The old tabs had the tap to open, then tap to select, but was just flaky.

We have added more options to the hamburger as a workaround. We could universally push everything that is in the selected tabs menu into the hamburger, which means you can still get everything but you still have to go to an intermediate page and then on that page open the hamburger, so better but far from ideal.

The hamburger menu is also slated for a rebuild, ie if you are at node X you'll be able to open it and then drill up and down and around and then select a node and only then actually open the new page. So I ideally want to keep the hamburger's concept of 'which node are you at' orthogonal to the tabs concept of 'what bits at this node are you looking at'.

**Comment by quaestor** [comment]
> The hamburger menu is also slated for a rebuild, ie if you are at node X you'll be able to open it and then drill up and down and around and then select a node and only then actually open the new page. So I ideally want to keep the hamburger's concept of 'which node are you at' orthogonal to the tabs concept of 'what bits at this node are you looking at'.

Right. That would increase the hamburger menu's value a lot; and then it doesn't make sense to put the other links in there, yes.
